app/src/main/res/layout/fragment_map.xml

```diff
@@ -11,6 +11,38 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:id="@+id/map"/>
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_marginVertical="10dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginRight="0dp"
+        android:layout_marginLeft="0dp"
+        android:elevation="10dp"
+        android:background="#fff"
+        android:id="@+id/relLayout1">
+
+        <ImageView
+            android:id="@+id/ic_magnify"
+            android:layout_width="wrap_content"
+            android:layout_height="40dp"
+            android:layout_centerVertical="true"
+            android:layout_marginLeft="5dp"
+            android:src="@drawable/ic_magnify"
+            />
+        <EditText
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_toRightOf="@id/ic_magnify"
+            android:layout_centerVertical="true"
+            android:textSize="15sp"
+            android:textColor="#000"
+            android:id="@+id/input_search"
+            android:hint="Enter adress"
```

Comment:
`android:hint="Enter adress"` should be using a string resource so that it's more easily translatable to Dutch and other languages. 

So it should be: `android:hint="@string/str_enter_adress"`, with in the file `res/values/strings.xml`: 
`<string name="str_enter_adress">Enter adress</string>`

Reply:
alright!